tdrs-backend/tdpservice/users/models.py

```diff
@@ -38,15 +38,21 @@ class User(AbstractUser):
         app_label="stts", model="region"
     )
 
-    location_id = models.PositiveIntegerField(null=True, blank=True)
-    location_type = models.ForeignKey(
-        ContentType,
-        on_delete=models.CASCADE,
+    stt = models.ForeignKey(
+        STT,
+        on_delete=models.deletion.SET_NULL,
+        related_name='users',
+        null=True,
+        blank=True
+    )
+
+    region = models.ForeignKey(
+        Region,
+        on_delete=models.deletion.SET_NULL,
+        related_name='users',
         null=True,
-        blank=True,
-        limit_choices_to=limit,
```

Comment:
If we are not using limit, we should delete it

Reply:
this is deleted code

Reply:
I meant to delete line 37

Reply:
ah, i see. good catch, i'll get it removed. thanks!